client/landing/gutenboarding/stores/onboard/selectors.ts

```diff
@@ -25,3 +26,7 @@ export const getSelectedDomain = ( state: State ) => state.domain;
 export const getSelectedSiteTitle = ( state: State ) => state.siteTitle;
 export const getDomainSearch = ( state: State ) =>
 	state.domainSearch || getSelectedSiteTitle( state );
+export const getIsFseSite = ( state: State ): boolean => {
```

Comment:
For boolean selectors, I think we have kind of a convention in Calypso to drop  the `get` prefix, and just start with `is` -- basically like the question they ask:

```suggestion
export const isFseSite = ( state: State ): boolean => {
```

Reply:
cool! now that I think of it, we can probably remove this selector since we can always use the design as the source of truth